{"name": "Process a phone order with Save and then Pay", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'ERROR IS Message: \\nStacktrace:\\nNoSuchElementError: An element could not be located on the page using the given sear...ata\\\\Roaming\\\\npm\\\\node_modules\\\\appium\\\\node_modules\\\\@appium\\\\base-driver\\\\lib\\\\basedriver\\\\commands\\\\find.ts:75:12)' == True\n +  where 'ERROR IS Message: \\nStacktrace:\\nNoSuchElementError: An element could not be located on the page using the given sear...ata\\\\Roaming\\\\npm\\\\node_modules\\\\appium\\\\node_modules\\\\@appium\\\\base-driver\\\\lib\\\\basedriver\\\\commands\\\\find.ts:75:12)' = order('Phone Order')\n +    where order = <logic.takeout_logic.Calculations object at 0x000001BF0B6895E0>.order\n +      where <logic.takeout_logic.Calculations object at 0x000001BF0B6895E0> = <test_phoneorder_card.TestPhoneOrderCard object at 0x000001BF0B57EF30>.phone", "trace": "self = <test_phoneorder_card.TestPhoneOrderCard object at 0x000001BF0B57EF30>\n\n    @allure.story(\"Phone Order Save and Pay\")\n    @allure.title(\"Process a phone order with Save and then Pay\")\n    def test_phoneorder_save_pay(self):\n    \n>       assert self.phone.order(\"Phone Order\") == True\nE       AssertionError: assert 'ERROR IS Message: \\nStacktrace:\\nNoSuchElementError: An element could not be located on the page using the given sear...ata\\\\Roaming\\\\npm\\\\node_modules\\\\appium\\\\node_modules\\\\@appium\\\\base-driver\\\\lib\\\\basedriver\\\\commands\\\\find.ts:75:12)' == True\nE        +  where 'ERROR IS Message: \\nStacktrace:\\nNoSuchElementError: An element could not be located on the page using the given sear...ata\\\\Roaming\\\\npm\\\\node_modules\\\\appium\\\\node_modules\\\\@appium\\\\base-driver\\\\lib\\\\basedriver\\\\commands\\\\find.ts:75:12)' = order('Phone Order')\nE        +    where order = <logic.takeout_logic.Calculations object at 0x000001BF0B6895E0>.order\nE        +      where <logic.takeout_logic.Calculations object at 0x000001BF0B6895E0> = <test_phoneorder_card.TestPhoneOrderCard object at 0x000001BF0B57EF30>.phone\n\nsuits\\test_orderflows\\test_card\\test_phoneorder_card.py:31: AssertionError"}, "start": 1741781703960, "stop": 1741781714465, "uuid": "59f6f1e7-0930-48f6-8014-c67ab63eeced", "historyId": "9979c322a3eae085832f011ab6e7b51b", "testCaseId": "9979c322a3eae085832f011ab6e7b51b", "fullName": "suits.test_orderflows.test_card.test_phoneorder_card.TestPhoneOrderCard#test_phoneorder_save_pay", "labels": [{"name": "story", "value": "Phone Order Save and Pay"}, {"name": "feature", "value": "Sanity Sheet"}, {"name": "parentSuite", "value": "suits.test_orderflows.test_card"}, {"name": "suite", "value": "test_phoneorder_card"}, {"name": "subSuite", "value": "TestPhoneOrderCard"}, {"name": "host", "value": "LAPTOP-KKMKONCC"}, {"name": "thread", "value": "4736-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "suits.test_orderflows.test_card.test_phoneorder_card"}]}